src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -104,6 +109,29 @@ private void setDataFormat(IList<DataFormatParameter> formatParameters)
          View.SetMappingSource(_mappings);
          ValidateMapping();
          InitializeErrorUnit();
+         setDimensionsForMappings();
+      }
+
+      private void setDimensionsForMappings()
+      {
+         foreach (var mapping in _mappings)
+         {
+            var mappingColumn = (mapping.Source as MappingDataFormatParameter)?.MappedColumn;
+            if (mappingColumn?.Unit == null) 
+               continue;
+
+            //initial settings for fraction dimension
+            if (mapping.ColumnInfo.DefaultDimension?.Name == Constants.Dimension.FRACTION && 
+                mappingColumn.Unit.ColumnName.IsNullOrEmpty() && 
```

Comment:
mappingColumn.Unit and mappingColumn.Unit?.. I think we need to be consistent in the same if...

Reply:
You are right, there is no way mappingColumn.Unit could be null here so no need for the check. Fixed.